docs/proposals/scheduledjob.md

```diff
@@ -0,0 +1,222 @@
+    Epsilon *int64
+
+    // ConcurrencyPolicy specifies how to treat concurrent executions of a Job.
+    ConcurrencyPolicy ConcurrencyPolicy
+
+    // Suspend flag tells the controller to suspend subsequent executions, it does
+    // not apply to already started executions.  Defaults to false.
+    Suspend bool
+
+    // Template is the object that describes the job that will be created when
+    // executing a ScheduledJob.
+    JobTemplate JobSpec
+
+    // Key of the selector that is added to Jobs to prevent concurrently running
+    // Jobs selecting their pods.
+    // Users can set this to an empty string to indicate that the system should
+    // not add any selector and label. If unspecified, system uses
+    // "scheduledjob.kubernetes.io/podTemplateHash".
+    // Value of this key is hash of ScheduledJobSpec.PodTemplateSpec.
+    // No label is added if this is set to empty string.
+    UniqueLabelKey string
+}
+
+// ConcurrencyPolicy describes how the job will be handled.
+// Only one of the following concurrent policies may be specified.
+// If none of the following policies is specified, the default one
+// is AllowConcurrent.
+type ConcurrencyPolicy string
+
+const (
+    // AllowConcurrent allows ScheduledJobs to run concurrently.
+    AllowConcurrent ConcurrencyPolicy = "Allow"
+
+    // ForbidConcurrent forbids concurrent runs, skipping next run if previous
+    // hasn't finished yet.
+    ForbidConcurrent ConcurrencyPolicy = "Forbid"
+
+    // ReplaceConcurrent cancels currently running job and replaces it with a new one.
+    ReplaceConcurrent ConcurrencyPolicy = "Replace"
+)
+```
+
+`ScheduledJobStatus` structure is defined to contain information about scheduled
+job executions.  The structure holds a list of currently running job instances
+and additional information about overall successful and unsuccessful job executions.
+
+```go
+// ScheduledJobStatus represents the current state of a Job.
+type ScheduledJobStatus struct {
+    Conditions []ScheduledJobCondition
+
+    // Successful tracks the overall amount of successful completions of this job.
+    Successful int64
+
+    // Failed tracks the overall amount of failures of this job.
+    Failed int64
+
+    // LastScheduleTime keeps information of when was the last time the job was successfully scheduled.
+    LastScheduleTime Time
+}
+
+type ScheduledJobConditionType string
+
+// These are valid conditions of a scheduledjob.
+const (
+    // ScheduledJobActive indicates the scheduled job is active and executes
+    // jobs according to the specified schedule.
+    ScheduledJobActive ScheduledJobConditionType = "Active"
+    // ScheduledJobSuspended means there won't be new jobs started,
+    // currently running jobs will continue to run to completion.
+    ScheduledJobSuspended ScheduledJobConditionType = "Suspended"
+)
+
+// ScheduledJobCondition describes current state of a job.
```

Comment:
You're right. I'll change the conditions array in status with an array of active job references.